cpp/msbuild/ice.cpp.props

```diff
@@ -122,14 +112,6 @@
     </Link>
   </ItemDefinitionGroup>
 
-  <ItemDefinitionGroup>
-    <Link>
-      <!-- We disable incremental linking (see above), which turns on /OPT:LBR, which in turn conflicts with -->
-      <!-- /EDITANDCONTINUE even though /OPT:LBR is an ARM-only option. So we turn off LBR. -->
-      <AdditionalOptions>/OPT:NOLBR %(AdditionalOptions)</AdditionalOptions>
```

Comment:
We don't use `EDITANDCONTINUE` so not clear why we have this comment. I also enable incremental linking which seems to be the defautl.